src/config/config.service.spec.ts

```diff
@@ -135,11 +135,6 @@ describe('ReconnectionConfigService', () => {
       await expect(setupConfigService({ QUEUE_HIGH_WATER: 'foo', ...env })).rejects.toBeDefined();
     });
 
-    it('missing provider account seed phrase should fail', async () => {
-      const { PROVIDER_ACCOUNT_SEED_PHRASE: dummy, ...env } = ALL_ENV;
-      await expect(setupConfigService({ PROVIDER_ACCOUNT_SEED_PHRASE: undefined, ...env })).rejects.toBeDefined();
-    });
-
```

Comment:
should replace this with a test that confirms the mutual exclusivity of seed phrase/eth private key